py-plugin/src/main/kotlin/org/jetbrains/research/deepbugs/python/ide/inspections/PyDeepBugsBinOperatorInspection.kt

```diff
@@ -29,6 +34,15 @@ class PyDeepBugsBinOperatorInspection : PyDeepBugsBinExprInspection() {
                 PyBinOp.extractOperatorText(node) ?: ""
             )
         }
+
+        override fun analyzeInspected(result: Float, node: NavigatablePsiElement, data: DataType) {
+            if (result > threshold && !PyDeepBugsConfig.shouldIgnore(data)) {
```

Comment:
Probably we should add some method like `DeepBugsPlugin.isProblem(result, data)` with this logic. Otherwise it is copy-pasted across whole project